guides/common/modules/proc_configuring-the-discovery-service.adoc

```diff
@@ -37,27 +37,41 @@ endif::[]
 hosts or virtual machines can be created with incorrect network configurations. On Dell servers, the new naming scheme can be disabled via `biosdevname=1` kernel command line option, for other hardware or virtual machines, the new naming scheme can be completely turned off via `net.ifnames=1`.
 endif::[]
 
-To use {ProjectServer} to provide the Discovery image, install the following RPM packages:
-
-* `tfm-rubygem-foreman_discovery`
-ifdef::satellite[]
-* `foreman-discovery-image`
-endif::[]
-* `tfm-rubygem-smart_proxy_discovery`
-
-The `tfm-rubygem-foreman_discovery` package contains the {Project} plug-in to handle discovered nodes, connections, and necessary database structures, and API.
-
+.Procedure
+. Install Discovery on {ProjectServer}:
++
 ifdef::satellite[]
```

Comment:
@maximiliankolb is this bit of text something you'd also want for Orcharhino?

Reply:
Yes. Please make it "if satellite or orcharhino"

Reply:
Addressed :+1: